README.md

```diff
@@ -166,6 +168,17 @@ simply `npm install` and `npm test`.
     provisions of this method, which you cannot possibly
     obtain in legacy engines.
 
+### Example of applying ES compatability shims in a browser project
+
+```html
+<script src="es5-shim.js"></script>
+<script src="es5-sham.js"></script>
+<script src="json3.js"></script>
```

Comment:
As `json3` is ECMA5.1, then it seems correct to go after `es5-shim`, I believe `es6-shim` also patches the ECMA5.1 JSON object, which won't happen if it were after all the es shims?